src/modules/iotjs_module_fs.c

```diff
@@ -202,7 +202,10 @@ JS_FUNCTION(Open) {
 }
 
 
-JS_FUNCTION(Read) {
+jerry_value_t fs_do_read_or_write(const jerry_value_t jfunc,
+                                  const jerry_value_t jthis,
+                                  const jerry_value_t jargv[],
+                                  const jerry_length_t jargc, bool do_read) {
```

Comment:
`do_read` should be in a new line, below the `jargc` parameter.

Reply:
In general, it's better to use custom types for `boolean` typed parameters, because that values are more informative. For example:

```c
typedef enum {
  IOTJS_FS_READ,
  IOTJS_FS_WRITE
} iotjs_fs_t;

jerry_value_t fs_do_read_or_write(const jerry_value_t jfunc,
                                  const jerry_value_t jthis,
                                  const jerry_value_t jargv[],
                                  const jerry_length_t jargc,
                                  const iotjs_fs_t fs_op);

fs_do_read_or_write(jfunc, jthis, jargv, jargc, IOTJS_FS_WRITE);
```

Reply:
clang-format wants `do_read` and `jargc` to be on the same line for some reason, style check was failing earlier because of it on Travis.